src/components/AddCard.tsx

```diff
@@ -1,33 +1,47 @@
 import React from 'react';
-import { Box, Pressable, useColorMode } from 'native-base';
-import { StyleSheet, ViewStyle } from 'react-native';
-import AddCardIcon from 'src/assets/images/addCardIcon.svg';
+import { Box, useColorMode } from 'native-base';
+import { StyleSheet, TouchableOpacity, ViewStyle } from 'react-native';
+import AddCardIcon from 'src/assets/images/add_white.svg';
 import Text from './KeeperText';
+import Colors from 'src/theme/Colors';
+import HexagonIcon from './HexagonIcon';
 
 type AddSignerCardProps = {
   name: string;
   callback?: (param: any) => void;
   cardStyles?: ViewStyle;
+  iconWidth?: number;
+  iconHeight?: number;
 };
 
-function AddCard({ name, callback, cardStyles }: AddSignerCardProps) {
+function AddCard({
+  name,
+  callback,
+  cardStyles,
+  iconWidth = 40,
+  iconHeight = 34,
+}: AddSignerCardProps) {
   const { colorMode } = useColorMode();
   return (
-    <Pressable
-      backgroundColor={`${colorMode}.pantoneGreenLight`}
-      borderColor={`${colorMode}.pantoneGreen`}
-      style={[styles.AddCardContainer, cardStyles && cardStyles]}
-      onPress={() => callback(name)}
-    >
-      <Box style={styles.detailContainer}>
-        <Box backgroundColor={`${colorMode}.pantoneGreen`} style={styles.iconWrapper}>
-          <AddCardIcon />
+    <TouchableOpacity onPress={() => callback(name)}>
+      <Box
+        backgroundColor={`${colorMode}.pantoneGreenLight`}
+        borderColor={`${colorMode}.pantoneGreen`}
+        style={[styles.AddCardContainer, cardStyles && cardStyles]}
```

Comment:
Can we verify if this doesn't evaluate to be true?

Reply:
It works as intended. Add Card for home screen has bigger width and height so we pass it in cardStyles prop. For other instances. It does not make any difference.